lnbits/static/js/wallet.js

```diff
@@ -697,10 +679,14 @@ window.app = Vue.createApp({
       this.$q.localStorage.set('lnbits.disclaimerShown', true)
     }
     // listen to incoming payments
-    LNbits.events.onInvoicePaid(this.g.wallet, payment => {
-      this.onPaymentReceived(payment.payment_hash)
+    LNbits.events.onInvoicePaid(this.g.wallet, data => {
+      console.log('Payment received:', data.payment.payment_hash)
+      console.log('Wallet balance:', data.wallet_balance)
+      console.log('Wallet ID:', this.g.wallet)
```

Comment:
do you want to keep these `console.log`?

Reply:
i like those logs, but we can remove it for now, but i think a nice frontend logger library would be cool, like `loglevel` https://github.com/pimterry/loglevel